### ISSUE TITLE
"Add To Playlist" menu doesn't show existing playlists

### ISSUE THREAD

**Issue by abarisain** [abarisain/dmix#723]
Apparently, a regression hit that menu : it only shows "(New Playlist)" and never the server known playlists

**Comment by avuton** [comment]
Yeah, I commented that in the code. That happened in the move from MainMenuActivity to NowPlayingActivity, if I remember correctly. Didn't write a bug, figured it was a known issue.

**Comment by avuton** [comment]
Oh, thinking of different bug, sorry.

**Comment by avuton** [comment]
Let me take a look at this one as well; last I saw this worked. Meh.
